src/models/item.ts

```diff
@@ -112,6 +112,7 @@ export class Item {
 
   isBeltable = false;
   isSackable = true;
+  isLockerable = true;
```

Comment:
This needs to be a function that's called when entering a locker. Otherwise we have a flag on nearly every item in the game. The only things that are not lockerable at this time are corpses and gold. Eventually, probably also twigs.